prime-angular-frontend/src/app/modules/enrolment/pages/pharmanet-enrolment-summary/pharmanet-enrolment-summary.component.ts

```diff
@@ -47,7 +47,7 @@ export class PharmanetEnrolmentSummaryComponent extends BaseEnrolmentPage implem
     private fb: FormBuilder
   ) {
     super(route, router);
-    this.showProgressBar = false;
+    this.showProgressBar = true;
```

Comment:
I don't think we want this to default show the progress bar.  The progress bar is only used on initial enrolment and in this case only displayed after you accept a TOA to show 100%.  It gets set based on route state data in ngOnInit that is only passed from the access agreement view.  By setting it in the constructor it is possible you'll see it a bounce in the screen content as it appears and then disappears when the view is revisited post acceptance.